py/desitarget/streams/cuts.py

```diff
@@ -0,0 +1,328 @@
+"""
+desitarget.streams.cuts
+=======================
+
+Target selection cuts for the DESI MWS Stellar Stream programs.
+
+Borrows heavily from Sergey Koposov.
+"""
+from time import time
+
+import healpy
+import datetime
+import os
+import scipy.interpolate
+import numpy as np
+import astropy.table as atpy
+
+from desitarget.cuts import _psflike
+from desitarget.streams.utilities import sphere_rotate, correct_pm, rotate_pm, \
+    betw, pm12_sel_func, plx_sel_func, get_CMD_interpolator, stream_distance,  \
+    get_stream_parameters
+from desitarget.streams.io import read_data_per_stream
+from desitarget.targets import resolve
+from desitarget.streams.targets import finalize
+
+# ADM set up the DESI default logger.
+from desiutil.log import get_logger
+log = get_logger()
+
+# ADM start the clock.
+start = time()
```

Comment:
It's a little unusual to start a clock ticking at the time of package import (vs. in a script or a function).  Is this really intended?

Reply:
Agreed. I moved the start inside the relevant function.